package.json

```diff
@@ -8,6 +8,7 @@
     "start": "next start"
   },
   "dependencies": {
+    "@vechain/connex": "^2.0.5",
```

Comment:
Generally I would advocate locking deps but it might be nice to auto update here, @KennethAshley who will we use for hosting (assume it's netlfiy - do you know if they build the site off of package-lock.json)?

Reply:
Yup that's exactly what we did 😄